test/watt_config.yml

```diff
@@ -0,0 +1,49 @@
+aggregate_prs_results_test:
+  workflow_name: "AggregatePRSResults"
+  test_name: "Base"
+  path: "/ImputationPipeline/AggregatePRSResults.wdl"
+  test_inputs: "/test/AggregatePRSResults/test_inputs.json"
+  expected_outputs: "/test/AggregatePRSResults/test_outputs.json"
+
+benchmark_vcfs_ignore_filters_test:
+  workflow_name: "BenchmarkVCFs"
+  test_name: "IgnoreFilters"
+  path: "/BenchmarkVCFs/BenchmarkVCFs.wdl"
```

Comment:
Have you tested running these locally with this directory setup? I think Watt is configured so that the location of the `yml` is the "root" when resolving these sorts of paths, so this file should live one directory up I think (unless I changed it to look up until it finds the root of the repo, which I was thinking about but think I skipped).

Reply:
It actually looks like you did implement a [search upwards for a root of a github repo](https://github.com/rickymagner/watt/blob/c01b17e6e1f2163802caf511a3c73d5afa365f1d/watt.py#L37) :)

however, the tests do still need to be run from the palantir-workflows root, because the paths in the input and expected json files are still interpreted (both by watt an cromwell) as relative to the current directory.  Would be pretty simple to connect that up to the `resolve_relative_path` functionality that is already there, but I don't think worth holding up.

opened [an issue](https://github.com/rickymagner/watt/issues/3) so we can maybe remember to do this is anyone has time

Reply:
Thanks for opening the issue. It sounds like the current limitation is just that the JSONs themselves have to use paths relative to the runtime directory, but the config file uses the repo root as `/`. I think I agree then that it would make sense to parse the JSON paths as also being relative to the repo root.